### PR TITLE
Registration Form

### PR DESCRIPTION
**Pull request by monocliffe**
Added a Registration form page.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -34,3 +34,5 @@
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
+.idea/
+.rakeTasks
```
Comment (tmwpengine) [inline .gitignore]: You can add the `.generators` file in here as well

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  def index
+    @users = User.order(:forename)
+  end
+
+  # GET /users/1
+  # GET /users/1.json
+  def show; end
+
+  # GET /users/new
+  def new
```
Comment (tmwpengine) [inline app/controllers/users_controller.rb:15]: Is this needed? I see a `create` function below that looks more useful

Comment (monocliffe) [reply app/controllers/users_controller.rb:15]: Most likely, no. It was auto-generated by rails and I never removed it.

File: .gitignore (hunk 1 above)
Comment (monocliffe) [reply .gitignore]: Done.

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (tmwpengine) [reply app/controllers/users_controller.rb:15]: No worries for now, things like this can be cleaned up down the line

**Review by tmwpengine — APPROVED** [review]
Looks good

**Pull request merged**